99_master-create-users-symlink.yaml

```diff
@@ -0,0 +1,60 @@
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
+metadata:
+  labels:
+    machineconfiguration.openshift.io/role: master
+  name: 99-openshift-machineconfig-master-users-symlink
+spec:
+  config:
+    ignition:
+      version: 3.4.0
+    systemd:
+      units:
+      - contents: |
+          [Unit]
+          Description=Allow systemd to create mount points on /
+          DefaultDependencies=no
+          
+          [Service]
+          Type=oneshot
+          ExecStart=chattr -i /
+          
+          [Install]
+          WantedBy=remote-fs-pre.target         
```

Comment:
there's also a `local-fs-pre.target` and `local-fs.target` since we are not mounting any remote partition like nfs, maybe we can use these target instead.. found them on: https://www.freedesktop.org/software/systemd/man/latest/systemd.special.html

Reply:
We need the file system to be immutable to create `/Users` before doing a virtio-fs mount. I don't know if systemd considers this local or remote, but an argument can be made that this is a remote filesystem since it lives on the host which is a different machine than the guest.

`local-fs-pre.target` feels like it would be too early since at this point we have no guarantee that the filesystem where we want to create the directory is already mounted.